apis/meta/conditions.go

```diff
@@ -75,48 +49,16 @@ const (
 	SuspendedReason string = "Suspended"
 )
 
-// DeepCopyInto is a deepcopy function, copying the receiver, writing into out. in must be non-nil.
-func (in *Condition) DeepCopyInto(out *Condition) {
-	*out = *in
-	in.LastTransitionTime.DeepCopyInto(&out.LastTransitionTime)
-}
-
-// DeepCopy is a deepcopy function, copying the receiver, creating a new Condition.
-func (in *Condition) DeepCopy() *Condition {
-	if in == nil {
-		return nil
-	}
-	out := new(Condition)
-	in.DeepCopyInto(out)
-	return out
-}
-
-// GetCondition returns the Condition from the given slice that matches the
-// given type.
-func GetCondition(conditions []Condition, conditionType string) *Condition {
-	for i := range conditions {
-		c := conditions[i]
-		if c.Type == conditionType {
-			return &c
-		}
-	}
-	return nil
-}
-
 // HasReadyCondition returns if the given Condition slice has a ReadyCondition
 // with a 'True' condition status.
-func HasReadyCondition(conditions []Condition) bool {
-	condition := GetCondition(conditions, ReadyCondition)
-	if condition == nil {
-		return false
-	}
-	return condition.Status == corev1.ConditionTrue
+func HasReadyCondition(conditions []metav1.Condition) bool {
```

Comment:
```suggestion
func InReadyCondition(conditions []metav1.Condition) bool {
```

The `func` name at present was suggested when I created the PR, but I did not take into account when I accepted the suggestion that it also checks if the `Ready==True`.